### PR TITLE
Display low trust/monitored/restricted users' messages

### PR DESCRIPTION
**Pull request by Azenet**
This PR adds a feature that displays information on messages that are from users that are manually monitored or banned across several channels with the new "shared ban information" feature. Additionally, messages from restricted users may be shown in chat for the moderator to see.

### CODE REVIEW

File: src/chatty/util/api/UserInfoManager.java [hunk 1]
```diff
@@ -80,6 +120,7 @@ public void resultReceived(Set<String> requested, Collection<UserInfo> result) {
             for (UserInfo info : result) {
                 notFound.remove(info.login);
                 perLogin.setResult(info.login, info);
+                perId.setResult(Long.valueOf(info.id), info);
```
Comment (Azenet) [inline src/chatty/util/api/UserInfoManager.java:123]: Doing this does not seem to have broken anything but I'd love some input on whether linking the 2 caches this way is an okay thing to do.

**Comment by tduva** [comment]
I can't comment on the actual changes yet since I haven't had time to go through it, but has there been some new information on the `low-trust-users` topic? Because last time I tried it the topic wasn't working with a Chatty token.

**Comment by tduva** [comment]
Twitch has finally officially documented the `low-trust-users` topic, so I'm looking at this again.

**Comment by Azenet** [comment]
I'd need to redo the code for this PR, I haven't had the time to look into it but the code I provided doesn't work at the moment.

**Comment by tduva** [comment]
Looks like my commit to resolve the conflicts caused all commits of master in between to be shown here as well, that wasn't what I intended. Sometimes I find GIt quite confusing. Either way, I squash merged it into master, so hopefully it's a clean commit in there at least. The feature seems to work now, but I'll refine some of it a bit more. Thanks for your help.

**Pull request merged**